torch/_inductor/select_algorithm.py

```diff
@@ -380,7 +380,10 @@ def generate(
             )
             if self.debug:
                 print("Generated Code:\n", code)
-            mod = PyCodeCache.load(code)
+            mod = PyCodeCache.load(
+                code,
+                extra=f"{kwargs['BLOCK_M'], kwargs['BLOCK_K'], kwargs['BLOCK_K']}-{num_stages}-{num_warps}-",
```

Comment:
Should not assume the kwargs only contains BLOCK_M/BLOCK_K.  

BLOCK_K is listed twice.

Reply:
Oh wow I missed that, one is meant to be BLOCK_N. Let me fix that real quick.